packages/slate-react/src/components/editable.tsx

```diff
@@ -1049,6 +1053,37 @@ export const Editable = (props: EditableProps) => {
   )
 }
 
+/**
+ * The props that get passed to renderPlaceholder
+ */
+export type RenderPlaceholderProps = {
+  contentEditable: boolean
+  children?: React.ReactNode
+  ref: React.RefObject<any>
```

Comment:
Can you put these `ref` and `contentEditable` keys inside an `attributes` hash? 

The pattern is essentially that there will be `children, attributes, ...stuff` where `attributes` must be mixed on to the component, and `children` is special-cased. (We might want to change this in the future for all of them at once, since it's a little strange, but let's go with consistency for now.)

Reply:
I assigned `children` the `any` prop, consistent with the other render prop interfaces, is that what you meant by:

> children is special-cased